src/language-js/index.js

```diff
@@ -10,7 +10,11 @@ const languages = [
     override: {
       since: "0.0.0",
       parsers: ["babylon", "flow"],
-      vscodeLanguageIds: ["javascript"]
+      vscodeLanguageIds: ["javascript"],
+      interpreters: ["node"]
```

Comment:
You don't need to override `interpreters`, [it's already there](https://github.com/ikatyang/linguist-languages/blob/v6.3.0/data/javascript.json#L38-L40).

Reply:
…I thought I’d deleted that line, but apparently I just moved `nodejs` to `extends`. As a great American said, d’oh!